### PR TITLE
Changed word in line 62. Updated milestone section

### PR DESCRIPTION
**Pull request by CBID2**
## Check List (Check all the applicable boxes) 

- [x ] My code follows the code style of this project.
- [x ] My change requires changes to the documentation.
- [ x] I have updated the documentation accordingly.
- [ x] All new and existing tests passed.
- [ x] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Hey Eddie! :) I followed your suggestions. Let me know if more improvements are needed. (Note: in lines 30-31 in my JSON file, I used the GitHub icon and its color because there is no other icon or color for scholarships and other awards.) 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2120"></a>

### CODE REVIEW

File: public/data/CBID2.json [hunk 1]
```diff
@@ -24,5 +24,13 @@
...
       "url": "https://github.com/users/CBID2/achievements/pull-shark"
     }
+    {
+      "title": "SheSharp New in Tech Scholarship Recepient",
+      "date":"October 2022",
+      "icon": "github",
+      "color": "grey",
+      "description":"Shesharp is a non-profit foundation that is dedicated to helping women and non-binary people start their careers in tech. I received this award as a result of expressing my interest in bridging the gaps between instructional design and coding in addtion to being an active member in SheSharp's Slack community.",
+      "url": "https://www.linkedin.com/posts/shesharp_announcing-our-new-in-tech-scholarship-winner-activity-6993866371780476928-q5-3?utm_source=share&utm_medium=member_desktop",
+    }
```
Comment (eddiejaoude) [inline public/data/CBID2.json:34]: Please see the GitHub Action inline comments about the invalid json

Also json file change should go to `main` branch (but the README changes should be to the `nextjs` branch)

Comment (CBID2) [reply public/data/CBID2.json:34]: Ok. How do I make sure the README file changes go into the mainJS?

Comment (eddiejaoude) [reply public/data/CBID2.json:34]: create a new branch from the default branch for each separate change

Comment (CBID2) [reply public/data/CBID2.json:34]: Still lost

**Comment by eddiejaoude** [comment]
Going to close PR as it is a duplicate of
https://github.com/EddieHubCommunity/LinkFree/pull/2124
https://github.com/EddieHubCommunity/LinkFree/pull/2123

**Pull request closed without merging**